airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -1026,7 +1045,10 @@ def execute(self, context: Context):
             if not self.table_project_id:
                 self.table_project_id = self.project_id
             else:
-                self.log.info("Ignoring project_id parameter, as table_project_id is found.")
+                self.log.info("Ignoring 'project_id' parameter, as 'table_project_id' is found.")
+
+        if self.job_id and self.table_id:
```

Comment:
What would happen if both are None 🤔 If exactly one is needed, there's a handle `exactly_one` in `airflow.utils.helpers`

Reply:
Thanks for showing me this method!
Fixed :)